packages/eslint-plugin/src/rules/no-empty-styled-expression/README.md

```diff
@@ -0,0 +1,59 @@
+# `no-empty-styled-expression`
+
+Discourages any `styled` expression to be used when passing empty arguments in `@compiled/react`.
+
+Passing an empty object or no object at all causes Compiled to build extra `div/span` elements, as opposed to simply using a `div`. This leads to reduced performance and is greatly discouraged. If a wrapper is necessary, opt to use a `div` or wrap it in the empty React fragment `<> <YourComponentHere></YourComponentHere> </>`.
+
+---
+
+## Rule details
+
+👎 Examples of **incorrect** code for this rule:
+
+```
+const Wrapper = styled.div();
+
+function Button() {
```

Comment:
The `Button` doesn't add any value to the example imo, should just be

```jsx
const EmptyStyledExpression = styled.div();
```

Same for all the others